composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": "^7.2.5 || ^8.0",
+        "php": "^7.3 || ^8.0",
```

Comment:
I _believe_ Takeout still works fine on PHP 7.2; it's only the tests that don't. As such, I'd rather we keep it at `7.2` since I still intend to fix the tests.

Thanks!